### DISCUSSION TITLE
THANK YOU!!

### DISCUSSION THREAD

**Discussion by jhmonroe** [danny-avila/LibreChat#4980] [General]
This project is amazing — you're amazing!
Thank you so much for doing this. It makes exploring the fast-evolving models in one place a pleasure.

**Comment by danny-avila** [comment]
You're welcome! 

If you'd like to give back to the project, I have a github sponsors page: https://github.com/sponsors/danny-avila

I also recommend the Code Interpreter API as a way to support the project + unlock a very useful feature: https://code.librechat.ai/pricing
